calculate/keywords.go

```diff
@@ -366,6 +366,67 @@ var languageToKeywords = map[Language][]string{
 		"with",
 		"yield",
 	},
+	"php": {
```

Comment:
where did you take this list from? are we sure nothing is missing?

Reply:
took it from here https://www.php.net/manual/en/reserved.keywords.php